docs/content/en/docs/Reference/auroraboot.md

```diff
@@ -0,0 +1,458 @@
+---
+title: "AuroraBoot"
+linkTitle: "AuroraBoot"
+weight: 8
+date: 2023-02-08
+description: >
+    Automatically provision machines with Kairos and AuroraBoot.
+---
+
+`AuroraBoot` is a tool designed to make the process of bootstrapping Kairos machines quick, simple and efficient. It is specifically designed for the Kairos operating system and provides a comprehensive solution for downloading required artifacts and provisioning a machine, both from network or manually via flashing to USB stick. With `AuroraBoot`, you can prepare the environment for network-based bootstrapping, download the necessary release assets, and also customize the installation media for USB-based mass-installations. Whether you're looking to install Kairos on a single machine or multiple machines, `AuroraBoot` makes it easy and efficient. 
+
+Auroraboot can be useful to:
+
+- prepare multiple-nodes in a lab before shipment
+- offer a simple, intuitive and streamlined way to deploy Kairos automatically and manually
+- deploy Kairos nodes in a network segment where we can already send workload to (running `AuroraBoot` in an already-existing downstream cluster)
+
+![AuroraBoot](https://user-images.githubusercontent.com/2420543/217617696-f993a8e3-55ac-4d3e-98f0-c2317cb54cb9.png)
+
+## Scope
+
+`AuroraBoot` has the following scope:
+
+- **Download** release assets in order to provision one or more machines
+- **Prepare** automatically the environment to boot from network
+- **Provision** machines from network with a version of Kairos and cloud config
+- **Customize** The installation media for installations from USB
+
+## Prerequisites
+
+- `docker` or a container engine of your choice
+- Port `8090`, `8080` and `67` free on the host running `AuroraBoot`
+- The machine running `AuroraBoot` have to be on the same network segment of the nodes to be bootstrapped
+- The nodes need to be configured to boot over network, or be capable of booting via USB for offline mode
+- `ProxyDHCP` supported by the `DHCP` network attempting to netboot (see also [pixiecore architecture](https://github.com/danderson/netboot/blob/master/pixiecore/README.booting.md#step-1-dhcpproxydhcp)).
+   There should be an already running `DHCP` server in your network. `AuroraBoot` doesn't take over the `DHCP` server, neither require you to do any specific configuration, however a `DHCP` server which is compliant to `ProxyDHCP` requests should be present in the same network running `AuroraBoot` and the machines to boot.
+
+## Overview
+
+To run `AuroraBoot`, simply use `docker` or the container engine of your choice (such as `podman`, ...). `Auroraboot` images are published in [quay](https://quay.io/repository/kairos/auroraboot) and the source code is available [in GitHub](https://github.com/kairos-io/AuroraBoot).
```

Comment:
we dont support running the binary?

Reply:
Not currently, we use the osbuilder-tool image /deps behind the scene